pkg/cmd/gist/create/create.go

```diff
@@ -91,13 +92,23 @@ func createRun(opts *CreateOptions) error {
 		fileArgs = []string{"-"}
 	}
 
-	files, err := processFiles(opts.IO.In, fileArgs)
+	files, nonGistFilesSorted, err := processFiles(opts.IO.In, fileArgs)
 	if err != nil {
 		return fmt.Errorf("failed to collect files for posting: %w", err)
 	}
 
+	processMessage := "Creating gist..."
+	completionMessage := "Created gist"
+	if len(files) > 1 && len(nonGistFilesSorted) >= 1 {
```

Comment:
I'm not sure the `len(nonGistFilesSorted) >= 1)` check is needed. Is it possible to create a gist with multiple files using only the piped command? If not, I am happy to remove this.